Source/Scene/ModelExperimental/ModelSceneGraph.js

```diff
@@ -601,16 +606,16 @@ ModelSceneGraph.prototype.configurePipeline = function (frameState) {
     modelPipelineStages.push(ModelClippingPlanesPipelineStage);
   }
 
+  if (model.hasSilhouette(frameState)) {
+    modelPipelineStages.push(ModelSilhouettePipelineStage);
+  }
+
   if (
     defined(model.splitDirection) &&
     model.splitDirection !== SplitDirection.NONE
   ) {
     modelPipelineStages.push(ModelSplitterPipelineStage);
   }
```

Comment:
I see that the splitter stage is not run if the model is used for classification. Is that behavior consistent with the old `Model`?

Reply:
`ClassificationModel` didn't have a `splitDirection` function. Plus it "automatically" works with clipping planes / split direction because it only draws on the rendered tileset parts.